pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/LogFileSequenceProcessor.java

```diff
@@ -91,20 +88,29 @@ public static LogFileSequenceProcessor create(
     }
 
     public void ingestLogFiles() throws Exception {
```

Comment:
Should we rename this as we're not actually ingesting in this method anymore?

Reply:
done